File: week3/numbers.c
#include <stdio.h>
#include <cs50.h>

// Linear search for number 0

int main(void) // main always returns an int
{
    int numbers[] = {2, 5, 8, 9, 3, 7, 0};

    for (int i = 0; i < 7; i++) 
    {
        if (numbers[i] == 0)
        {
            printf("Found\n");
            return 0; // means all went well & exits program
        }
    }
    printf("Not found\n");
    return 1; // means something went wrong & exits program
}

File: week6/numbers.py
# Linear search in a list

from cs50 import get_int
import sys

# List of numbers
numbers = [4, 6, 8, 2, 7, 5, 0]

# Prompt user for a number
n = get_int("Which number are you looking for? ")

if n in numbers:
    print("Found.")
    sys.exit(0)
else:
    print("Not found.")
    sys.exit(1)